### PR TITLE
Structs

### PR DESCRIPTION
**Pull request by vindarel**

### CODE REVIEW

File: data-structures.md [hunk 1]
```diff
@@ -1264,6 +1264,182 @@ we remove an element with `remf`.
 ;; => "foo!!!"
 ~~~
 
+## Structures
+
+Structures offer a way to store data in named slots. They support
+single inheritance.
+
+They can be considered super-seeded by classes of the Common Lisp
```
Comment (pnathan) [inline data-structures.md]: superceded

File: data-structures.md [hunk 2]
```diff
@@ -1264,6 +1264,182 @@ we remove an element with `remf`.
...
+Object System (CLOS), that don't have their limitations (see below).
```
Comment (pnathan) [inline data-structures.md]: Also, DEFMETHOD doesn't dispatch on DEFSTRUCT (iirc - might need to test that assertion, but it's what I recollect).  DEFMETHOD is slower than DEFUN though, so... ymmv.

And, structs don't really interop with clos too well.

Comment (vindarel) [reply data-structures.md]: Thanks. I don't feel like talking about defmethod here (it wasn't introduced yet), 

> And, structs don't really interop with clos too well.

I invite you to rephrase what can be added :)

Comment (pnathan) [reply data-structures.md]: that's fine.

**Comment by pnathan** [comment]
:+1:

**Pull request merged**